runtime/litmus/src/weights/mod.rs

```diff
@@ -17,12 +17,12 @@
 #![allow(clippy::unnecessary_cast)]
 
 pub mod cumulus_pallet_xcmp_queue;
-pub mod frame_system;
+// pub mod frame_system;
 pub mod pallet_asset_manager;
 pub mod pallet_balances;
 pub mod pallet_bridge;
 pub mod pallet_bridge_transfer;
-pub mod pallet_collator_selection;
+// pub mod pallet_collator_selection;
```

Comment:
why are these commented out here and in the other `*/weights/mod.rs` files? can they be removed?

Reply:
I can't remember exactly why it was commented out.

I can uncomment them, it doesn't matter too much tho:
- the weightInfo uses `()`
- litmus will be removed soon anyway

Reply:
I removed it completely.